pkg/utils/status.go

```diff
@@ -767,6 +844,14 @@ func observabilityStatusCheck(ctx context.Context, instance *csmv1.ContainerStor
 	metricsRunning := false
 	topologyRunning := false
 
+	driverName := instance.Spec.Driver.CSIDriverType
+
+	// TODO: PowerScale DriverType should be changed from "isilon" to "powerscale"
```

Comment:
Remove TODO

Reply:
Why? It is a current TODO

Reply:
We aren't suppose to have TODO in checkin code

Reply:
Hmm ok didn't realize that -- where is the documentation on that, or is it just an assumed standard? How do we document something like this that we are planning on fixing in an upcoming release?